packages/renderer/src/lib/image/BuildImageFromContainerfile.svelte

```diff
@@ -88,6 +114,79 @@ async function buildContainerImage(): Promise<void> {
   }
 }
 
+// Function to handle the building of container images for multiple platforms and creating a manifest
+// afterwards
+async function buildMultiplePlatformImagesAndCreateManifest(): Promise<void> {
+  buildFinished = false;
+
+  // Collection of build IDs, this is needed for being able to create the manifest
+  // as we need to know either the IDs or the names of the images that were built
+  buildIDs = [];
+
+  if (containerFilePath && selectedProvider) {
+    // Extract the relative path from the containerFilePath and containerBuildContextDirectory
+    const relativeContainerfilePath = containerFilePath.substring(containerBuildContextDirectory.length + 1);
+
+    // We'll iterate over each platform and build the image
+    for (const platform of platforms) {
```

Comment:
I am wondering if we should not have this logic in the main, rather than the frontend ?

Reply:
yeah, you are right, this should be instead in the main code. i'm going to mark this as draft and move the code there instead.

Reply:
@axel7083 changing my opinion, sorry.

After going through the code, I think it's good that keep this in the svelte file / user facing due to our extensive use of streaming data back for buildImage and having multiple builds / creating manifest after would create a large amount of refactoring if the code was more in main rather than user facing.

Because of my limited bandwidth and the 13 other issues i've opened / fixed to get this feature added, I won't be able to implement a more thorough implementation in time for next week's release, so we may have to do this PR temporary (to fix the non-functioning building) until we can do a larger refactor IMHO.

This PR can be a temporary fix until we have a more permanent one / have a function to do multiple builds + create manifest.

I've opened up issue: https://github.com/containers/podman-desktop/issues/6815

Reply:
I understand the bandwidth issue, and the PR LGTM so I will approve it